src/platform/library/lib/trace.c

```diff
@@ -12,8 +12,7 @@
 #include <sof/trace/dma-trace.h>
 
 /* enable trace by default in testbench */
-int test_bench_trace = 1;
-int debug;
+int host_trace_level = 1;
```

Comment:
should be merged with the previous commit to support bisection. Also wouldn't it be better to use `LOG_LEVEL_ERROR`?

Reply:
ack fo bisect.